policyengine_us/variables/gov/states/mt/tax/income/mt_capital_gains_tax_joint.py

```diff
@@ -0,0 +1,37 @@
+from policyengine_us.model_api import *
+
+
+class mt_capital_gains_tax_joint(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana net long-term capital gains tax"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    def formula(person, period, parameters):
+        is_head = person("is_tax_unit_head", period)
+        capital_gains = person("long_term_capital_gains", period)
+        taxable_income = person("mt_taxable_income_joint", period)
+        nonqualified_income = taxable_income - capital_gains
+        p = parameters(period).gov.states.mt.tax.income.main.capital_gains
+        rate_p = parameters(
+            period
+        ).gov.states.mt.tax.income.main.capital_gains.rates.reduced_capital_gains
+        filing_status = person.tax_unit("filing_status", period)
+        gap = p.threshold[filing_status] - nonqualified_income
+        lower_base_tax = gap * rate_p.lower[filing_status]
+        reduced_capital_gains_tax = where(
+            capital_gains <= gap,
+            capital_gains * rate_p.lower[filing_status],
+            (
```

Comment:
simplify the where statements to 
```
capital_gains_under_threshold = capital_gains <= gap
lower_capital_gains_tax = capital_gains * rate_p.lower[filing_status]
```
Same think for the last argument - break up into individual variables for each calculation step